auth_backend/backends.py

```diff
@@ -34,10 +34,25 @@ def authenticate(self, email=None, username=None, password=None, **kwargs):
         if strategy:
             payload['strategy'] = strategy
 
+        auth_api_client = AuthApiClient(cas_credentials)
         status, data = auth_api_client.call('sessions', 'POST', payload)
```

Comment:
for interest sake, you could make this a callable class if you used the `__call__` magic function ... then you could just 
call client like

`auth_api_client('sessions', 'POST', payload)`